roles/init_dbserver/defaults/main.yml

```diff
@@ -43,6 +43,8 @@ pg_initdb_options: "-k --encoding=UTF-8 --locale=en_US.UTF-8"
 pg_unix_socket_directories:
   - "/var/run/postgresql"
 
+# initial configuration for starting postgres
+pg_init_conf_params: []
```

Comment:
Can you please add an example (commented) of how to define settings here?

Reply:
Done